batect.yml

```diff
@@ -85,12 +85,12 @@ tasks:
     run:
       container: run
       command: ./cli/build/install/ort/bin/ort requirements
-  funTest:
+  funTestAnalyzer:
     description: >-
-      A task to execute ORT's functional tests in the runtime container. Can be limited to e.g. the "analyzer" project
-      like `./batect funTest -- -p analyzer`.
+      A task to execute ORT's functional tests for the analyzer in the runtime container.
     prerequisites:
       - buildFunTest
     run:
       container: run
-      command: ./gradlew funTest
+      command: "'./gradlew funTest jacocoFunTestReport -p analyzer'"
```

Comment:
Could you amend you commit message about why this nested quoting with double *and* single-quotes is required? Have you verified that it does not work with either type of quoting only?

Reply:
Ah, I just saw this is nicely explained at https://batect.dev/docs/reference/config/tasks/#command. Would you mind adding that link to the commit message?

Reply:
PS: I also raised the [question](https://github.com/batect/batect/discussions/1355) whether the current Batect behavior is intended. My hunch is that it's not, and Batect could be smarter about adding additional task arguments to inside of quotes in a `command`.